crates/oxc_regular_expression/src/display.rs

```diff
@@ -0,0 +1,541 @@
+        (r"/(cg)(?<n>cg)(?:g)/", None, true),
+        (r"/^(?=ab)\b(?!cd)(?<=ef)\B(?<!gh)$/", None, true),
+        (r"/^(?<!ab)$/", None, true),
+        (r"/[abc]/", None, true),
+        (r"/[a&&b]/v", None, true),
+        (r"/[a--b]/v", None, true),
+        (r"/[^a--b--c]/v", None, true),
+        (r"/[a[b[c[d[e[f[g[h[i[j[k[l]]]]]]]]]]]]/v", None, true),
+        (r"/[\q{abc|d|e|}]/v", None, true),
+        (r"/\p{Basic_Emoji}/v", None, true),
+        (r"/[|\]]/", None, true),
+        (r"/c\]/", None, true),
+        ("/a{0}|b{1,2}|c{3,}/i", None, true),
+        (r"/Em🥹j/", None, true),
+        (r"/\n\cM\0\x41\./", None, true),
+        (r"/\n\cM\0\x41\u1234\./u", None, true),
+        (r"/[\bb]/", None, true),
+        (r"/\d+/gu", None, true),
+        (r"/\D/gu", None, true),
+        (r"/\w/gu", None, true),
+        (r"/\w+/gu", None, true),
+        (r"/\s/g", None, true),
+        (r"/\s+/g", None, true),
+        (r"/\t\n\v\f\r/", None, true),
+        // we lose the flags ordering
+        ("/abcd/igv", Some("/abcd/giv"), true),
+        // we lose the flags ordering
+        (r"/\d/ug", Some(r"/\d/gu"), true),
+        // NOTE: we capitalize all hex values.
+        (r"/\n\cM\0\x41\u{1f600}\./u", Some(r"/\n\cM\0\x41\u{1F600}\./u"), true),
+        // TODO: tests below do **NOT** parse with unicode mode on
+        (r"/c]/", None, false),
+        // Octal tests from: <https://github.com/tc39/test262/blob/d62fa93c8f9ce5e687c0bbaa5d2b59670ab2ff60/test/annexB/language/literals/regexp/legacy-octal-escape.js>
+        (r"/\1/", None, false),
+        (r"/\2/", None, false),
+        (r"/\3/", None, false),
+        (r"/\4/", None, false),
+        (r"/\5/", None, false),
+        (r"/\6/", None, false),
+        (r"/\7/", None, false),
+        // NOTE: we remove leading zeroes
+        (r"/\00/", Some(r"/\0/"), false),
+        // NOTE: we remove leading zeroes
+        (r"/\07/", Some(r"/\7/"), false),
+        (r"/\40/", None, false),
+        (r"/\47/", None, false),
+        (r"/\70/", None, false),
+        (r"/\77/", None, false),
+        // NOTE: we remove leading zeroes
+        (r"/\000/", Some(r"/\0/"), false),
+        // NOTE: we remove leading zeroes
+        (r"/\007/", Some(r"/\7/"), false),
+        // NOTE: we remove leading zeroes
+        (r"/\070/", Some(r"/\70/"), false),
+        (r"/\300/", None, false),
+        (r"/\307/", None, false),
+        (r"/\370/", None, false),
+        (r"/\377/", None, false),
+        (r"/(.)\1/", None, false),
+        // Identity escape from: <https://github.com/tc39/test262/blob/d62fa93c8f9ce5e687c0bbaa5d2b59670ab2ff60/test/annexB/language/literals/regexp/identity-escape.js>
+        (r"/\C/", None, false),
+        (r"/O\PQ/", None, false),
+        (r"/\8/", None, false),
+        (r"/7\89/", None, false),
+        (r"/\9/", None, false),
+        (r"/8\90/", None, false),
+        // TODO: fails to parse
+        // (r"(.)(.)(.)(.)(.)(.)(.)(.)\8\8", None, false),
```

Comment:
This always raises an error even with Unicode mode off.

Reply:
```suggestion
        // (r"/(.)(.)(.)(.)(.)(.)(.)(.)\8\8/", None, false),
```

`/` is missing...? 🙈